{"Functions": {"RejectedExecutionException(java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#Throwable_E@", "Throwable: the cause (which is saved for later retrieval by the @B_java#lang#Throwable#getCause_E@ method)"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a RejectedExecutionException with the specified cause. The detail message is set to (cause == null ? null : cause.toString()) (which typically contains the class and detail message of cause).", "history": "Added in API level 1", "FullName": "public RejectedExecutionException (Throwable cause)"}, "RejectedExecutionException(java.lang.String,java.lang.Throwable)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message"], ["@B_java#lang#Throwable_E@", "Throwable: the cause (which is saved for later retrieval by the @B_java#lang#Throwable#getCause_E@ method)"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a RejectedExecutionException with the specified detail message and cause.", "history": "Added in API level 1", "FullName": "public RejectedExecutionException (String message, Throwable cause)"}, "RejectedExecutionException()": {"Returns": [], "Parameters": [], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a RejectedExecutionException with no detail message. The cause is not initialized, and may subsequently be initialized by a call to @B_java#lang#Throwable#initCause_E@.", "history": "Added in API level 1", "FullName": "public RejectedExecutionException ()"}, "RejectedExecutionException(java.lang.String)": {"Returns": [], "Parameters": [["@B_java#lang#String_E@", "String: the detail message"]], "Throws": [], "SeeAlso": [], "Permissions": [], "Description": "Constructs a RejectedExecutionException with the specified detail message. The cause is not initialized, and may subsequently be initialized by a call to @B_java#lang#Throwable#initCause_E@.", "history": "Added in API level 1", "FullName": "public RejectedExecutionException (String message)"}}, "Inheritance": ["@B_java#lang#Throwable_E@", "@B_java#lang#Exception_E@", "@B_java#lang#RuntimeException_E@"], "ClassName": "java.util.concurrent.RejectedExecutionException", "ClassDesc": "Exception thrown by an @B_java#util#concurrent#Executor_E@ when a task cannot be accepted for execution."}